<ui version="4.0" >
 <author>G. Allen Morris III</author>
 <comment>Dialog to add a new project.</comment>
 <class>AddProject</class>
 <widget class="QDialog" name="AddProject" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AddProject</string>
  </property>
  <property name="whatsThis" >
   <string>What is this.</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="Layout1" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>180</y>
     <width>476</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="buttonHelp" >
      <property name="text" >
       <string>&amp;Help</string>
      </property>
      <property name="shortcut" >
       <string>F1</string>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="Horizontal Spacing2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOk" >
      <property name="text" >
       <string>&amp;OK</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel" >
      <property name="text" >
       <string>&amp;Cancel</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="projectName" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>50</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;p align="right">Project Name&lt;/p></string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
   <property name="buddy" >
    <cstring>projectName</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel4" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>20</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;p align="right">Parent&lt;/p></string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;p align="right">Project description&lt;/p></string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
   <property name="buddy" >
    <cstring>projectDescription</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="Parent_Project" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>20</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip" >
    <string>The Parent project.</string>
   </property>
   <property name="whatsThis" >
    <string comment="This is replaced with the parent project name." >The parent project</string>
   </property>
   <property name="text" >
    <string>Parent Project</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="projectDescription" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>80</y>
     <width>351</width>
     <height>94</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>projectName</tabstop>
  <tabstop>projectDescription</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonHelp</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>AddProject</receiver>
   <slot>add_project()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>349</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddProject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>430</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHelp</sender>
   <signal>clicked()</signal>
   <receiver>AddProject</receiver>
   <slot>show_help()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>31</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
